app/views/collection_drafts/forms/_descriptive_keywords.html.erb

```diff
@@ -186,3 +186,13 @@
 <script type="text/javascript">
   setupScienceKeywords(<%= raw @science_keywords.to_json %>);
 </script>
+
+<!-- Passing regex for validation of Value and Data Type fields -->
+<script>
+  var floatRegexString = "<%= Regexp.quote  CollectionDraftsController::FLOAT_REGEX %>";
```

Comment:
nitpick: 1) you probably don't need another script tag, I think you can just stick it in the existing script tag above; 2) if a standalone script tag I would think you need the type attr as the above tag has